core/variant/binder_common.h

```diff
@@ -224,6 +224,14 @@ struct PtrToArg<char32_t> {
 template <typename T>
 struct VariantObjectClassChecker {
 	static _FORCE_INLINE_ bool check(const Variant &p_variant) {
+#ifdef DEBUG_ENABLED
+		bool freed;
+		Object *object = p_variant.get_validated_object_with_check(freed);
+		if (freed) {
```

Comment:
This will only catch if the variant is an object, and it was freed, not if it's `null` to start with, or not an object

Reply:
I think `can_convert_strict()` already ensures that.